### ISSUE TITLE
lld: error: undefined hidden symbol: ReadRefImageIntoSharedMemory

### ISSUE THREAD

**Issue by katastrice** [openphotogrammetry/meshroomcl#8]
MultiviewStereoCL  failes with:

```
PatchMatch::Run
---------------
OpenCL platform : AMD Accelerated Parallel Processing 
OpenCL device   : gfx1031 
Error -11: Failed to build program OpenCL kernels!
Program Build Log:
lld: error: undefined hidden symbol: ReadRefImageIntoSharedMemory
>>> referenced by C:\Users\blaa\AppData\Local\Temp\comgr-59b14c\input\linked.bc.o:(ComputeInitialCostKernel)
>>> referenced by C:\Users\blaa\AppData\Local\Temp\comgr-59b14c\input\linked.bc.o:(ComputeInitialCostKernel)
Error: Creating the executable from LLVM IRs failed.
```

on:

Ryzen 5800X
B550
AMD 6700XT

**Comment by revisionarian** [comment]
Hi @katastrice, thanks a lot for reporting the issue that you encountered with MeshroomCL. We were able to replicate the problem; it was caused by a bug in our software that affects RDNA2 graphics cards with the latest AMD drivers (21.6.1).  To solve the issue, we have released a new version of MeshroomCL, version 0.7.0, available here:

https://github.com/openphotogrammetry/meshroomcl/releases

Please give this new version of MeshroomCL a try, and let us know if it does not fix the problem. Thanks for your feedback!

**Comment by katastrice** [comment]
Thank you very much, I can confirm with v0.7.0 it works just perfect.
